app/server.go

```diff
@@ -215,8 +215,10 @@ func NewServer(options ...Option) (*Server, error) {
 		}
 	})
 
-	mlog.Info(fmt.Sprintf("Current version is %v (%v/%v/%v/%v)", model.CurrentVersion, model.BuildNumber, model.BuildDate, model.BuildHash, model.BuildHashEnterprise))
-	mlog.Info(fmt.Sprintf("Enterprise Enabled: %v", model.BuildEnterpriseReady))
+	logCurrentVersion := fmt.Sprintf("Current version is %v (%v/%v/%v/%v)", model.CurrentVersion, model.BuildNumber, model.BuildDate, model.BuildHash, model.BuildHashEnterprise)
```

Comment:
I would like see also see this part converted to structured logging. This would help understanding the meaning of the field. What do you think about moving this log messages into `app/server_app_adapters.go:58`. This way they would also be shown as the fist message, which is handy.

Reply:
I didn't converted it because they were informative messages during the start-up and are thought for being human readable. I would like to have others input here like @jwilander and @lieut-data

Reply:
I can see arguments both ways -- given that we're unlikely to chart these numbers / parameters, I might lean towards leaving it human readable for now. Or, even duplicating with structured parameters, given this is a low-volume log.

Reply:
I like the idea of keep both, structured logging and human readable.

Reply:
Done